### ISSUE TITLE
Issue with code highlighting in Goland 2019.3.1

### ISSUE THREAD

**Issue by xelarion** [one-dark/jetbrains-one-dark-theme#119]
The problem is the same as in pycharm：

> I found some further info, seems the color scheme of the theme is somehow not persistent. This is what I did: 
> 
> 1. Open PyCharm: The colors are a mix between One Dark theme and Darcula.
> 2. Uninstall One Dark theme and restart PyCharm.
> 3. Install One Dark theme: The colors are displayed correctly. 
> 4. After closing and reopening PyCharm, the colors are changed back to a mix between One Dark theme and Darcula.
> 
> What do you suggest? I would very much like to fix this issue so I can enjoy your beautiful theme again. Thank you very much for your help and for your stellar work.
> 

_Originally posted by @visini in https://github.com/one-dark/jetbrains-one-dark-theme/issues/115#issuecomment-568863798_

**Comment by mskelton** [comment]
Thanks for reporting this.  I'll do my best to look into this over the weekend.

**Comment by dewey** [comment]
Just confirming that I have the same issue, opened a bug at Jetbrains and they referred me here.

```
GO-193.5662.65, JRE 11.0.5+10-b520.17x64 JetBrains s.r.o, OS Mac OS X(x86_64) v10.15.2, screens 1920x1080, 1440x900; Retina
```

**Comment by petrjahoda** [comment]
Solution I found: after installing the theme do this:

Go to Editor-> Color Scheme, export the theme.
Restart program
After that, go back to Editor -> Color Scheme and select that exported theme and restart
After that, both UI and editor is in one dark mode

**Comment by petrjahoda** [comment]
Also... I found out, that program UI stays in One Dark, but only Editor coloring changes to Darcula, that solution described makes it work for Editor.

**Comment by mskelton** [comment]
This should be fixed in [v3.3.3](https://github.com/one-dark/jetbrains-one-dark-theme/releases/tag/v3.3.3).  If not, please let me know.

**Comment by petrjahoda** [comment]
Tested right now. Have to uninstall the theme via Plugins, restart and install again via Plugins.
But after that... it works like a charm.
Thank you very much.
